app/controllers/AuthenticationController.scala

```diff
@@ -429,6 +431,91 @@ class AuthenticationController @Inject()(
     }
   }
 
+  private lazy val oidcClientId = conf.WebKnossos.OIDC.clientId
+  private lazy val oidcProviderUrl = conf.WebKnossos.OIDC.providerURL
+
+  lazy val oidcConfig: OpenIdConnectConfig = OpenIdConnectConfig(oidcProviderUrl, oidcClientId)
+  lazy val absoluteOidcCallbackURL = "http://localhost:9000/api/auth/oidc/callback"
+
+  // Claims from https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
+  case class OpenConnectId(iss: String,
```

Comment:
I’d move this to the OpenIdConnectClient file as well, and import it here if needed

Reply:
Agreed